.github/workflows/ci.yml

```diff
@@ -41,6 +41,9 @@ jobs:
           pip install -U coveralls
       - name: Run style-checks
         uses: pre-commit/action@v3.0.1
```

Comment:
the pre-commit action fails in CI, please fix it. Don't explicitly add another step below to run mypy